src/how-can-i-help.md

```diff
@@ -74,7 +74,7 @@ Before contributing to Sugar Labs, you should use Sugarizer or Sugar Desktop.
 ## <a name="OUTREACHY-GSOC"></a> Coding, Documentation and Quality Assurance
 Coding, documentation and quality assurance is important - it is required everywhere from producing new software, rolling out new features, maintaining and upgrading current releases, distributing instructions for others to know how to operating the software, and ensuring that others can use software with any problems.
 
-  * We have mentoring opportunities through our participation in programs such as Outreachy, [Google Summer Of Code](https://wiki.sugarlabs.org/go/Summer_of_Code/2017), and [Google Code-In](https://wiki.sugarlabs.org/go/Google_Code_In_2017). You can read about these programs in the Wiki.
+  * We have mentoring opportunities through our participation in programs such as Outreachy, [Google Summer Of Code](https://wiki.sugarlabs.org/go/Summer_of_Code/2018), and [Google Code-In](https://wiki.sugarlabs.org/go/Google_Code_In_2019). You can read about these programs in the Wiki.
```

Comment:
Thanks for making these changes, the Google Summer of Code page should lead [here](https://github.com/sugarlabs/GSoC) and the GCI page should be removed as GCI has been stopped.

Reply:
I have made the changes as suggested by you! :)